integration_tests/cork_test.go

```diff
@@ -112,8 +112,8 @@ func (s *IntegrationTestSuite) TestCork() {
 					CellarId: cellarId,
 					CallData: &SubmitRequest_AaveV2Stablecoin{
 						&AaveV2Stablecoin{
-							Function: &AaveV2Stablecoin_ClaimAndUnstake{
-								&ClaimAndUnstake{},
+							Function: &AaveV2Stablecoin_ClaimAndUnstake_{
+								&AaveV2Stablecoin_ClaimAndUnstake{},
```

Comment:
Now that we have this integration test functioning well, perhaps we should add some of the more complicated calls that have some arguments to ensure things are working as we expect.